app/js/views/tag-list.jsx

```diff
@@ -1,25 +1,16 @@
 import React from 'components/react';
 
-import TagItem from 'js/views/tag-item';
-
 export default class TagList extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.foxbox = props.foxbox;
-  }
-
   render() {
-    let tagNodes = this.props.tags.map((tag) => (
-        <TagItem
-          key={tag.id}
-          id={tag.id}
-          name={tag.name}
-          checked={tag.checked}
-          serviceId={this.props.serviceId}
-          foxbox={this.foxbox}/>
-      )
-    );
+    let tagNodes = this.props.tags.map((tag) => {
+      return (<li key={tag} className="tag-list__item">
```

Comment:
nit: what about using the short notation for arrow functions with a return here?